RecoLocalTracker/SiPhase2VectorHitBuilder/test/Clusters_productionAndTesting.py

```diff
@@ -0,0 +1,103 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
consider if all of these configs (which will almost certainly stop working a few minutes after they're committed, assuming they even work now) are really necessary for the central repository

Reply:
Good point. I will remove those and we will replace them with proper validation configs once we move the validation code to a proper package.